### ISSUE TITLE
Array to string conversion

### ISSUE THREAD

**Issue by andrey-helldar** [GeneaLabs/laravel-model-caching#128]
# Issue
```
ErrorException (E_NOTICE)
Array to string conversion
```

`$where`:
```
array:4 [
  "type" => "In"
  "column" => "cars_model_uuid"
  "values" => array:1 [
    0 => array:1 [
      "uuid" => "5b937720-a7bd-11e4-80e1-00155d030302"
    ]
  ]
  "boolean" => "and"
]
```

## Environment
Laravel Version: *5.6.23*
Laravel Model Caching Package Version: *0.2.57*
PHP Version: *7.2.0*
Homestead Version: *7.6.0*
Operating System & Version: *Windows 10 Pro x64*

## Stack Trace
![2018-05-24 17-54-05 whoops there was an error - google chrome](https://user-images.githubusercontent.com/10347617/40493520-95a2650c-5f7b-11e8-80f0-8b5125d493a7.png)

## Query
```
$models_uuids = CarsModel::whereCarsClassUuid($class->uuid)
    ->whereNotIn('uuid', $this->excluded_models)
    ->get(['uuid']);

return Car::query()
    ->with('carModel.carClass.carMark')
    ->whereIn('cars_model_uuid', $models_uuids)
    ->orderBy('price')
    ->paginate($this->per_page);
```

**Comment by andrey-helldar** [comment]
Yes, I saw my mistake that I did not call the pluck method, but without it the query worked when caching was turned off.

@mikebronner Sorry, I mixed the line with the version. 2.7.1 is the version of `doctrine/dbal` :)

**Comment by mikebronner** [comment]
Thanks for the update. I will create a test-case using the queries you provided above.

**Comment by mikebronner** [comment]
I've been able to reproduce the error, working on a fix now.

**Comment by mikebronner** [comment]
This should now be fixed in version 0.2.58. Please let me know how it works for you.

**Comment by andrey-helldar** [comment]
There is no mistake. All perfectly. Thanks!
